<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterCreator_CareerBeliefPage</class>
 <widget class="QWizardPage" name="CharacterCreator_CareerBeliefPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QLabel" name="lblCharacterDetail">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>90</y>
     <width>205</width>
     <height>238</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameBadge">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>15</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblCharacterName">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>25</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>镜子猎人</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBelif">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>369</y>
     <width>208</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>348</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>信仰</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>12</y>
     <width>196</width>
     <height>379</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsCG">
      <property name="maximumSize">
       <size>
        <width>196</width>
        <height>296</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gravatar是Globally Recognized Avatar的缩写，是gravatar推出的一项服务，意为“全球通用头像”。如果在Gravatar的服务器上放置了你自己的头像，只要提供与之关联的email地址，coeus便可以通过该email获取到你的个性头像，并且呈现给其它在线玩家。&lt;/p&gt;&lt;p&gt;访问：&lt;a href=&quot;http://cn.gravatar.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://cn.gravatar.com/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gravatar是Globally Recognized Avatar的缩写，是gravatar推出的一项服务，意为“全球通用头像”。如果在Gravatar的服务器上放置了你自己的头像，只要提供与之关联的email地址，coeus便可以通过该email获取到你的个性头像，并且呈现给其它在线玩家。&lt;/p&gt;&lt;p&gt;访问：&lt;a href=&quot;http://cn.gravatar.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://cn.gravatar.com/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(85, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>性  别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optFemale">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>女</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../bin/client/images/female.png</normaloff>../../../bin/client/images/female.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>15</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optMale">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>男</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../bin/client/images/male.png</normaloff>../../../bin/client/images/male.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>18</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="cmbAvatarList">
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
